packages/clicker-ui/src/Action.js

```diff
@@ -0,0 +1,11 @@
+class Action {
```

Comment:
Actions should be kept in the game package. These need to be replayable in both ui and server. Also it seems you added mutation tracking here (moneyAdded, incomeGained). These mutations should be handled in the `getCurrentState`